doxygen/01_Installation.dox

```diff
@@ -65,22 +65,19 @@ repository.
      For Windows, download Microsoft Visual Studio Community Edition from
      \a https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx.
      When installing Visual Studio, one should select the 'Desktop 
-    development with C++' configuration' and the 'Windows 8.1 SDK' and 'Windows 
+    development with C++ configuration' and the 'Windows 8.1 SDK' and 'Windows 
```

Comment:
the quotes were wrong before but they are now wrong in a different way - it should read

>  'Desktop development with C++' configuration

Reply:
Corrected and updated these changes.